packages/wrangler/src/api/integrations/platform/index.ts

```diff
@@ -111,7 +111,6 @@ export async function getPlatformProxy<
 	const miniflareOptions = await run(
 		{
 			FILE_BASED_REGISTRY: Boolean(options.experimentalRegistry),
-			DEV_ENV: false,
```

Comment:
anything we need to dbl check with the frameworks folks here? A bit surprised this still defaults to `false` here 🤔

Reply:
This is just removing dead code—the `DEV_ENV` value here was never used to actually enable/disable SDW (there's a `TODO: actually use` comment in `experimental-flags.ts`)